lib/active_job/queue_adapters/cloudtasker_adapter.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# ActiveJob docs: http://guides.rubyonrails.org/active_job_basics.html
+# Example adapters ref: https://github.com/rails/rails/tree/master/activejob/lib/active_job/queue_adapters
+
+module ActiveJob
+  module QueueAdapters
+    # == Cloudtasker adapter for Active Job
+    #
+    # To use Cloudtasker set the queue_adapter config to +:cloudtasker+.
+    #
+    #   Rails.application.config.active_job.queue_adapter = :cloudtasker
+    class CloudtaskerAdapter
+      SERIALIZATION_FILTERED_KEYS = [
+        'executions', # Given by the worker at processing
+        'provider_job_id', # Also given by the worker at processing
+        'priority' # What is priority?
```

Comment:
`priority` is used by certain backends (e.g. Backburner) to prioritize jobs within a queue. You can write that this option in the context of cloudtasker as priorities are managed through queues.

Reply:
OK! (never used a backend that had this option :()
So It isn't used, I'm removing this comment - and keep filtering the 'priority' option out from the payload.